projects/Mallard/src/components/front/collection-page.tsx

```diff
@@ -139,6 +142,13 @@ const CollectionPage = ({
                                 issue,
                                 front,
                             }}
+                            style={{
```

Comment:
Can we add these to stylesheets?

Reply:
yes!

Reply:
Merci beaucoup

Reply:
Ping me a re-request when they're done and I'll +1